### DISCUSSION TITLE
[Feature Request]: Add title for datatable

### DISCUSSION THREAD

**Discussion by amshehzad** [rappasoft/laravel-livewire-tables#1481] [Feature Requests]
### Overview

Would be nice to have an option for having a title of datatable ( like a card header )

**Comment by lrljoe** [comment]
Assigned to you as you expressed an interest in doing this on Discord

**Comment by lrljoe** [comment]
Where do you see this living in relation to the table/toolbar/etc

**Comment by lrljoe** [comment]
https://discord.com/channels/1118369537155399690/1164021047943757824

**Comment by amshehzad** [comment]
> Assigned to you as you expressed an interest in doing this on Discord

Yeah I'll do it later this week

**Comment by lrljoe** [comment]
@amshehzad is initially assigned this task
